doc/neps/nep-0007-datetime-proposal.rst

```diff
@@ -14,6 +14,7 @@ Revised only slightly from the third proposal by
 :Contact: ivan@selidor.net
 :Date: 2008-07-30
 
+:Status: Final
```

Comment:
Maybe this should go up at the top? This is a weird one b/c the author/contact stuff just above is referring to another NEP...